### PR TITLE
Migration plugin docs to integration framework

### PR DESCRIPTION
**Pull request by nywilken**
Add scaffolding for [HashiCorp integrations](https://github.com/hashicorp/integrations)

### CODE REVIEW

File: docs/builders/lxd.mdx [hunk 1]
```diff
@@ -26,6 +26,24 @@ builder does not work with LXC.
 
 Below is a fully functioning example.
 
+### HCL
```
Comment (BrandonRomano) [inline docs/builders/lxd.mdx:29]:
```suggestion
**HCL2**
```

File: docs/builders/lxd.mdx [hunk 2]
```diff
@@ -26,6 +26,24 @@ builder does not work with LXC.
...
+    description = "Trivial repackage with Packer"
+  }
+}
+
+build {
+  sources = ["lxd.lxd-xenial"]
+}
+```
+
+### JSON
```
Comment (BrandonRomano) [inline docs/builders/lxd.mdx:45]:
```suggestion
**JSON**
```

**Review by BrandonRomano — COMMENTED** [review]
Think we wanted to roll w/ HCL / JSON titles as bold.  Mostly just suggesting in following convention with the others.

p.s. I think if you accept these changes you'll have to rebuild.

**Review by lbajolet-hashicorp — APPROVED** [review]
LGTM!

**Pull request merged**